src/newcharacter.cpp

```diff
@@ -580,6 +580,15 @@ bool avatar::create( character_type type, const std::string &tempname )
         cash = rng( 1500000, 2000000 );
     }
 
+    if( has_trait( trait_id( "XS" ) ) ) {
+        set_stored_kcal( 10000 );
```

Comment:
Maybe jsonize it?

Reply:
Not a bad idea. I'd need a bit of time to do that, but we ain't in a hurry.